src/cil.mli

```diff
@@ -2549,6 +2549,7 @@ val unoblivType : typ -> typ
 
 val isOblivFunc : typ -> bool
 val isOblivInt : typ -> bool
+val isOblivFloat : typ -> bool (*CHANGE*)
```

Comment:
That line actually did not "change to" its current form; it was freshly added. In this case I think a comment explaining the change isn't quite necessary as it is a simple line. I will remove the comment. (See my remark below on the other CHANGE comment).